bin/package_build.py

```diff
@@ -249,6 +250,24 @@ def rockylinux():
 		DATA.close()
 		print(f"Saved!\nfilename: {q}")
 
+def zOSOpenTools():
+	url = "https://api.github.com/users/ZOSOpenTools/repos"
+	try:
+		req = requests.get(url)
+		git_data = json.loads(req.content)
+		if req.status_code == 404:
+			raise Exception("404 File not found")
+	except Exception as e:
+		print("Couldn't pull. Error: ",str(e))
+	
+	zOS_list = []
+	for repo in git_data:
+		repo_info = {"packageName":repo["name"],"description":repo["description"],"version":""}
+		zOS_list.append(repo_info)
+	
+	with open('ZOSOpenTools.json','w') as file:
+		json.dump(zOS_list,file,indent=2)
```

Comment:
Saving it directly will erase all the previous data in the json. You should compare the new data with the old one and append it to the end of the file.

Reply:
We actually do just clobber old versions of the files :laughing: so this is in keeping with the rest of the tool. I think it's OK, but something we should address more holistically. Can you create an issue for this?

Reply:
I think its OK for now , we need to keep in mind the changes we are going to add in the future for the SQL based database design and I dont think any issue will be required for now .